src/pages/evm/staking/StakingTab.vue

```diff
@@ -0,0 +1,254 @@
+
+const label = 'current';
+
+const { t: $t } = useI18n();
+const uiDecimals = 2;
+const ant = getAntelope();
+const chainStore = useChainStore();
+const chainSettings = chainStore.currentChain.settings as EVMChainSettings;
+const userStore = useUserStore();
+const balanceStore = useBalancesStore();
+const accountStore = useAccountStore();
+
+const systemToken = chainSettings.getSystemToken();
+const systemTokenSymbol = systemToken.symbol;
+const systemTokenDecimals = systemToken.decimals;
+const stakedToken = chainSettings.getStakedSystemToken();
+const stakedTokenSymbol = stakedToken.symbol;
+const stakedTokenDecimals = stakedToken.decimals;
+
+
+// data
+const oneEth = ethers.BigNumber.from('1'.concat('0'.repeat(systemTokenDecimals)));
+const inputModelValue = ref(ethers.constants.Zero);
+const estimatedGas = ref(ethers.constants.Zero);
+// computed
+const stakedRatio = computed(() => chainStore.getStakedRatio(label));
+const outputModelValue = computed(() => {
+    if (stakedRatio.value.isZero()) {
+        return ethers.constants.Zero;
+    }
+    const output = inputModelValue.value.mul(stakedRatio.value).div(oneEth);
+    return output;
+});
+const fiatLocale = computed(() => userStore.fiatLocale);
+const fiatCurrency = computed(() => userStore.fiatCurrency);
+const systemTokenBalanceInfo = computed(() => balanceStore.currentBalances.filter(
+    balance => balance.token.contract === systemToken.address)[0],
+);
+const systemTokenFiatPrice = computed(() => systemTokenBalanceInfo.value?.token.price.getAmountInFiatStr(1) ?? '1');
+const systemTokenBalance = computed(() => systemTokenBalanceInfo.value?.amount ?? ethers.constants.Zero);
+const sidebarContent = computed(() => {
+    const header = $t('evm_stake.stake_sidebar_title', { symbol: systemTokenSymbol });
+    const content = [{
+        text: $t(
+            'evm_stake.stake_sidebar_content_fragment_1',
+            { systemSymbol: systemTokenSymbol, stakedSymbol: stakedTokenSymbol },
+        ),
+    }, {
+        text: $t('evm_stake.stake_sidebar_content_fragment_2_bold'),
+        bold: true,
+    }, {
+        text: $t(
+            'evm_stake.stake_sidebar_content_fragment_3',
+            { systemSymbol: systemTokenSymbol, stakedSymbol: stakedTokenSymbol },
+        ),
+    }, {
+        text: $t('evm_stake.stake_sidebar_content_fragment_4_bold'),
+        bold: true,
+    }, {
+        text: $t(
+            'evm_stake.stake_sidebar_content_fragment_5',
+            { systemSymbol: systemTokenSymbol, stakedSymbol: stakedTokenSymbol },
+        ),
+    }];
+
+    return {
+        header,
+        content,
+    };
+});
+const availableTostake = computed(() => {
+    const available = systemTokenBalance.value.sub(estimatedGas.value);
+
+    if (available.lt(0)) {
+        return ethers.constants.Zero;
+    }
+    return available;
+});
+const formIsValid = computed(() =>
+    !outputModelValue.value.isZero() &&
+    inputModelValue.value.lt(availableTostake.value),
```

Comment:
this should be less than or equal right?

Reply:
correct again